edge-util/src/Microsoft.Azure.Devices.Edge.Util/uds/HttpBufferedStream.cs

```diff
@@ -77,6 +77,31 @@ public async Task<string> ReadLineAsync(CancellationToken cancellationToken)
             }
         }
 
+        public string ReadLine()
+        {
+            int position = 0;
+            var buffer = new byte[1];
```

Comment:
As discussed, please see if it's easier to not have a loop that reads one byte at a time. (Especially for the `Async` version since it will probably generate a lot of `Task` garbage for every `await` unless the `Stream` implementation happens to use `ValueTask`.)

Reply:
BufferedStream does the buffering necessary.. so anything else is not very useful, and adds unnecessary complexity. As discussed, I will keep this as it is.